src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/IfError.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT license.
 
 using System.Collections.Generic;
+using System.Collections.ObjectModel;
```

Comment:
necessary using?

Reply:
ah, nevermind.  I see the readonlycollection

Reply:
Yeah - for ReadOnlyCollection

Reply:
But MS.PowerFx.Core.Binding was not, so removed it :-)

Reply:
Removed